### PR TITLE
[MIG] base_search_fuzzy: Migration to 11.0

### PR DESCRIPTION
**Pull request by cubells**
cc @Tecnativa

### CODE REVIEW

File: base_search_fuzzy/tests/test_query_generation.py [hunk 1]
```diff
@@ -37,8 +36,8 @@ def test_fuzzy_where_generation(self):
             where_clause_params)
         self.assertEqual(
             complete_where,
-            'SELECT FROM "res_partner" WHERE '
-            '("res_partner"."name" % \'test\')')
+            b'SELECT FROM "res_partner" WHERE '
```
Comment (pedrobaeza) [inline base_search_fuzzy/tests/test_query_generation.py]: I think the `b` (binary) is not needed

File: base_search_fuzzy/models/ir_model.py [hunk 2]
```diff
@@ -43,7 +43,7 @@ def decorate_leaf_to_sql(self, eleaf):
             if left in model._fields:
                 params = str(right)
 
-            if isinstance(params, basestring):
+            if isinstance(params, pycompat.string_types):
```
Comment (pedrobaeza) [inline base_search_fuzzy/models/ir_model.py]: Is this something made by `2to3`? I think `isinstance(params, str)` is better

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: base_search_fuzzy/models/ir_model.py (hunk 2 above)
Comment (yajo) [reply base_search_fuzzy/models/ir_model.py]: Indeed, there's no need to support py2 for v11.

**Review by yajo — APPROVED** [review]

File: base_search_fuzzy/tests/test_query_generation.py (hunk 1 above)
Comment (cubells) [reply base_search_fuzzy/tests/test_query_generation.py]: It's necessary because complet_where is binary.

Comment (pedrobaeza) [reply base_search_fuzzy/tests/test_query_generation.py]: Uhm, I don't think so. @Yajo what do you think?

Comment (yajo) [reply base_search_fuzzy/tests/test_query_generation.py]: I have no reasons to believe he's wrong, and Travis was green, so it's OK for me.

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Please squash before merging

**Comment by cubells** [comment]
@pedrobaeza done!

**Pull request merged**